packages/api-plugin-fulfillment-method-shipping-ups/README.md

```diff
@@ -0,0 +1,70 @@
+# api-plugin-fulfillment-method-shipping-ups
+
+[![npm (scoped)](https://img.shields.io/npm/v/@reactioncommerce/api-plugin-fulfillment-method-shipping-ups.svg)](https://www.npmjs.com/package/@reactioncommerce/api-plugin-fulfillment-method-shipping-ups)
+[![CircleCI](https://circleci.com/gh/reactioncommerce/api-plugin-fulfillment-method-shipping-ups.svg?style=svg)](https://circleci.com/gh/reactioncommerce/api-plugin-fulfillment-method-shipping-ups)
+
+
+## Summary
```

Comment:
As I have mentioned earlier, I think we should change the name of this plugin to method-shipping-dynamic-rate since this **DOES NOT** implement the UPS shipping and I don't want anybody getting their hopes up.

Reply:
As we discussed, I had left the name as UPS in case we were planning to enhance the plugin to make it more realistic UPS. But since we decided it is not going to be top priority, I shall make the changes to rename this plugin to 
`api-plugin-fulfillment-method-shipping-dynamic-rate`